packages/send/frontend/src/apps/send/pages/ClosePage.vue

```diff
@@ -1,8 +1,16 @@
 <script lang="ts" setup>
+import { SIGN_IN_COMPLETE } from '@send-frontend/lib/const';
+
 // Try window.close every 3 seconds to close the window after authentication
 setInterval(() => {
   window.close();
   console.log('Attempting to close window...');
+
+  window.postMessage(
+    { type: SIGN_IN_COMPLETE },
+    window.location.origin
+  );
+
```

Comment:
Closing the tab after authentication, part 1/5

When we reach the page that wants to close the tab, we send a message to the bridge, saying it's OK to close.

If we do this any sooner in the process, we do not finish getting the auth data.